core/src/test/scala/kafka/server/RemoteLeaderEndPointTest.scala

```diff
@@ -58,7 +58,7 @@ class RemoteLeaderEndPointTest {
         blockingSend = new MockBlockingSender(offsets = new util.HashMap[TopicPartition, EpochEndOffset](),
             sourceBroker = sourceBroker, time = time)
         endPoint = new RemoteLeaderEndPoint(logPrefix, blockingSend, fetchSessionHandler,
-            config, replicaManager, QuotaFactory.UnboundedQuota, () => MetadataVersion.MINIMUM_KRAFT_VERSION)
+            config, replicaManager, QuotaFactory.UnboundedQuota, () => MetadataVersion.MINIMUM_KRAFT_VERSION, () => 1)
```

Comment:
nit: Should we add a small unit test in this file to ensure that the value is propagated correctly?

Reply:
Done.